docs/cloud-tutorials/interactive-timetravel.md

```diff
@@ -37,7 +37,7 @@ chmod +x debug-proxy
 </Tabs>
 
 :::info
-The DBOS time travel proxy securely connects to the provenance database, an append-only replica of your application database maintained by DBOS Cloud.
+The DBOS time travel proxy securely connects to the [provenance database](../api-reference/provenance-database.md), an append-only replica of your application database maintained by DBOS Cloud.
```

Comment:
This is an improvement, but I question (pre-existing) use of the word "replica".  Representation?  There's not a great existing word I can find for a query-optimized change log.

Reply:
I used "replica" here to try to clarify that we're not adding storage burdens to the main app database (the original PG time travel implementation simply kept all versions in the same app DB and impacted app performance badly). In our case, it is like a read replica but with all previous versions in it.